templates/vpc.yaml

```diff
@@ -1,6 +1,13 @@
 AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
 Description: Amazon S3 Find and Forget VPC (uksb-1qjminsd5)
 
+Globals:
+  Function:
+    Runtime: python3.8
```

Comment:
can we use 3.9?

Reply:
Yes, we can and we should. I'll change it.